### DISCUSSION TITLE
1.21.4?

### DISCUSSION THREAD

**Discussion by LCSOGthb** [EssentialsX/Essentials#5987] [Q&A] [answered]
When the mod support Minecraft latest version 1.21.4, because the mod only support up to 1.20.1 only.

**Comment by Evidentsinger14** [comment, marked as answer]
EssentialsX is a bukkit plugin, not a mod. and Latest development builds support 1.21.4
https://essentialsx.net/downloads.html

**Reply by LCSOGthb** [reply]
Thanks!

**Reply by Navaden** [reply]
What about a stable release? i have a minecraft server on aternos and dev builds aren't listed on there, i've been waiting for a new release for a better part of the past year. Older version of the plugin breaks all commands and doesn't work. The server is now on 1.21.4.

**Reply by Evidentsinger14** [reply]
Stable builds will be released when they are stable enough to be released. 
Use a different host that allows you to upload your own jars if you need it
